Makefile.am

```diff
@@ -152,6 +152,35 @@ JSLINT_TARGET = jslint
 endif WITH_JSLINT
 lint: acilint apilint $(POLINT_TARGET) $(PYLINT_TARGET) $(JSLINT_TARGET)
 
+.PHONY: patchcheck
+patchcheck: all
```

Comment:
This is a [major nitpick](https://i.imgflip.com/1mam2x.jpg) but could it rather be called `develcheck`? The two `ch` in close succession don't quite roll of my keyboard, if that's not an intention.